src/main/java/com/gooddata/lcm/LcmEntitiesDeserializer.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2007-2018, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.lcm;
+
+import com.gooddata.collections.PageableListDeserializer;
+import com.gooddata.collections.Paging;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Deserializer of {@link LcmEntities}.
+ */
+class LcmEntitiesDeserializer extends PageableListDeserializer<LcmEntities, LcmEntity> {
```

Comment:
What about to add these (de)serializers as a static inner classes of LcmEntities?

Reply:
done